src/main/java/org/opensearch/knn/indices/Model.java

```diff
@@ -42,7 +41,7 @@
      * @param modelBlob binary representation of model template index. Can be null if model is not yet in CREATED state.
      * @param modelID model identifier
      */
-    public Model(ModelMetadata modelMetadata, @Nullable byte[] modelBlob, @NonNull String modelID) {
```

Comment:
Basically, dropping `@NonNull` since this the "default", nullable arguments are marked as `@Nullable`

Reply:
This makes sense.